src/components/shared/Extensions/Accordion/partials/SpecialPercentageInput.tsx

```diff
@@ -3,32 +3,36 @@ import React, { FC } from 'react';
 import SpecialInput from '~common/Extensions/SpecialInput';
 import FormError from '~shared/Extensions/FormError';
 import { SpecialInputProps } from '../types';
+import ConnectForm from './ConnectForm';
 
 const displayName = 'Extensions.SpecialPercentageInput';
 
 const SpecialPercentageInput: FC<SpecialInputProps> = ({
   defaultValue,
   maxValue,
   minValue,
-  register,
   errors,
   name = '',
 }) => (
-  <div className="text-right">
-    <div className="flex justify-end flex-col w-[8.875rem]">
-      <SpecialInput
-        defaultValue={defaultValue as number}
-        register={register}
-        isError={!!errors?.[name]?.message}
-        name={name}
-        min={minValue}
-        max={maxValue}
-        type="percent"
-        placeholder="1"
-      />
-      {errors?.[name] && <FormError>{errors?.[name]?.message}</FormError>}
-    </div>
-  </div>
+  <ConnectForm>
+    {({ register }) => (
+      <div className="text-right">
```

Comment:
Are you sure `text-right` is required here?
You can pass the `alignment` prop to `FormError` to adjust the text to the right.